dojo/tools/twistlock/parser.py

```diff
@@ -6,6 +6,8 @@
 import textwrap
 from datetime import datetime
 
+import dateutil.parser
```

Comment:
```suggestion
import dateutil.parser
from django.utils import timezone
```
I would be surprised if ruff is happy with the import order with this new line. be prepared to have to run ruff again